luckyapi/modloader/patches/Pop-up.gd

```diff
@@ -5,132 +5,6 @@ onready var modloader: Reference = get_tree().modloader
 func option_is_findable(type):
     return modloader.can_find_symbol(type)
 
-func add_cards(f_rarities):
```

Comment:
Although this does make it possible to start a run, it isn't a solution; this code is here for a reason. Its removal has likely broken multiple things, but at minimum it makes modded symbols that are marked as "unfindable"— only obtainable through interactions— findable.

Reply:
You're right, it did make use of option is findable, it seems declaring the comfy_pillow_trigger on ready makes it start fine and all symbols work with adjacents